root/files/resource/Sql/sqlserver/Create_UserStore.sql

```diff
@@ -0,0 +1,130 @@
+USE [master]
+GO
+
+ALTER DATABASE [UserStore] SET SINGLE_USER

```

Comment:
@daisukenishino 

If *UserStore* database does not exist, `ALTER DATABASE` statement and `DROP DATABASE` statement generate errors. So, check for the existence of *UserStore* database.

Please revise as follows:

```sql
IF EXISTS (select * from sysdatabases where name='UserStore')
BEGIN
	ALTER DATABASE [UserStore] SET SINGLE_USER
	DROP  DATABASE [UserStore]
END
GO
```